samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf

```
@@ -13,6 +13,9 @@ CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
 CONFIG_LTO=y
 
 CONFIG_BT=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
```

Ditto, observer implied by central
```suggestion
CONFIG_BT_PERIPHERAL=y
CONFIG_BT_CENTRAL=y
```

---

Ok.